src/services/document_service.py

```diff
@@ -183,7 +183,7 @@ def save(
                 node.get_context_storage_attribute(),
                 parent_id=parent_uid,
             )
-            return {"_id": node.uid, "type": node.entity["type"], "name": node.name}
+            return {"type": SIMOS.LINK, "ref": node.uid, "targetType": node.entity["type"], "targetName": node.name}
```

Comment:
```suggestion
            return {"type": SIMOS.LINK.value, "ref": node.uid, "targetType": node.entity["type"], "targetName": node.name}
```
Should it be like so?